package.json

```diff
@@ -263,7 +263,7 @@
     "@metamask/message-manager": "^7.3.0",
     "@metamask/metamask-eth-abis": "^3.0.0",
     "@metamask/name-controller": "^3.0.0",
-    "@metamask/network-controller": "^12.2.0",
+    "@metamask/network-controller": "^15.0.0",
```

Comment:
v13 has big breaking changes. The changes necessary to adopt v13 are here: https://github.com/MetaMask/metamask-extension/pull/20652

Reply:
I know I said it might but I'm not sure if this is required for assets-controllers version bump?